src/epplibwrapper/utility/pool.py

```diff
@@ -85,6 +85,19 @@ def _keepalive(self, c):
             logger.error(message, exc_info=True)
             raise PoolError(code=PoolErrorCodes.KEEP_ALIVE_FAILED) from err
 
+    def _keepalive_periodic(self):
+        delay = float(self.keepalive) / self.size
+        while 1:
+            try:
+                with self.get() as c:
+                    self._keepalive(c)
+            except PoolError as err:
+                logger.error(err.message, exc_info=True)
+            except self.exc_classes:
+                # Nothing to do, the pool will generate a new connection later
+                pass
+            gevent.sleep(delay)
+
```

Comment:
Remember to add a comment to this explaining that we're overriding it from the connpool library and the reasoning behind it

Reply:
Good catch, done